jfrog-cli/jfrog/buildinfo_test.go

```diff
@@ -195,56 +197,71 @@ func TestArtifactoryCleanBuildInfo(t *testing.T) {
 	cleanArtifactoryTest()
 }
 
-func TestCollectGitBuildInfo(t *testing.T) {
+func TestBuildAddGit(t *testing.T) {
 	initArtifactoryTest(t)
 	gitCollectCliRunner := tests.NewJfrogCli(execMain, "jfrog rt", "")
```

Comment:
Can we use **artifactoryCli** instead of creating this **gitCollectCliRunner**?

Reply:
Not for this test. The build-add-git command doesn't allow passing **-url**, **-user** and **-password** flags which are passed when using the **artifactoryCli**.